libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -38,39 +36,22 @@ export const AccountManager = ({
     },
     [gridRef]
   );
-  const { data, error, loading } = useDataProvider<AccountFields[], never>({
+  const { data } = useDataProvider<AccountFields[], never>({
     dataProvider: aggregatedAccountsDataProvider,
     update,
     variables,
   });
   if (!dataRef.current && data) {
     dataRef.current = data;
   }
-  const getRows = async ({
-    successCallback,
-    startRow,
-    endRow,
-  }: GetRowsParams) => {
-    const rowsThisBlock = dataRef.current
-      ? dataRef.current.slice(startRow, endRow)
-      : [];
-    const lastRow = dataRef.current?.length ?? -1;
-    successCallback(rowsThisBlock, lastRow);
-  };
   return (
-    <AsyncRenderer loading={loading} error={error} data={data}>
-      {data && (
-        <AccountTable
-          rowModelType={data?.length ? 'infinite' : 'clientSide'}
-          rowData={data?.length ? undefined : []}
-          ref={gridRef}
-          datasource={{ getRows }}
-          onClickAsset={onClickAsset}
-          onClickDeposit={onClickDeposit}
-          onClickWithdraw={onClickWithdraw}
-        />
-      )}
-    </AsyncRenderer>
+    <AccountTable
```

Comment:
Data in this table will not be updated. It will always render data returned by initial value of { data } from useDataProvider hook.
What is the reason behind removing rowModelType and datasource ?

Reply:
This is the reason why I removed it. 
![Screenshot 2022-10-10 at 13 14 17](https://user-images.githubusercontent.com/16125548/195129166-34f285d0-7877-41dd-bcfc-a47cd7580d23.png)

And how exactly does it work, because if I add getRows, rowModelType and dataSource back it seems that I have no accounts.  is this because I am doing `rowData={data?.length ? undefined : []}`
<img width="1080" alt="Screenshot 2022-10-11 at 16 10 42" src="https://user-images.githubusercontent.com/16125548/195129866-f1bbdf5f-2a79-4e47-93c5-2eab2cdf7927.png">

Reply:
```
rowModelType={data?.length ? 'infinite' : 'clientSide'}
rowData={data?.length ? undefined : []}
```

We switch between rowModelTypes because
 - infinite is much easier to do update without re rendering 
 - clientSide because ag-grid do not support overlayNoRowsTemplate when its in infinite rowModelType
 🤷